### PR TITLE
LS-461-first-draft

### PR DESCRIPTION
**Pull request by mendonk**
Basics from doc. Needs more.

### CODE REVIEW

File: modules/ROOT/pages/heartbeat-vm.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
+= Heartbeat on VM/Server
+
+This document describes how to install Pulsar Heartbeat on a virtual machine (VM) or server. For installation with the Docker image, see xref:quickstart-helm-installs.adoc[Helm Chart Installation].
+
+== Install Heartbeat Binary
+
+The release page is at https://github.com/datastax/pulsar-heartbeat/releases. The latest version is 1.0.12, which matches with the Git release tag and Docker image.
```
Comment (cdbartholomew) [inline modules/ROOT/pages/heartbeat-vm.adoc:7]: Can we remove the explicit version number so that we don't have to keep this updated?

**Review by cdbartholomew — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/heartbeat-vm.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+This document describes how to install Pulsar Heartbeat on a virtual machine (VM) or server. For installation with the Docker image, see xref:quickstart-helm-installs.adoc[Helm Chart Installation].
+
+== Install Heartbeat Binary
+
+Download the heartbeat binary `.gz` file matching your OS and process architecture from the https://github.com/datastax/pulsar-heartbeat/releases[releases page]. Use the Linux commands `uname -m` and `uname -o` to determine your process architecture.
+
+For example, if I run the command `uname -m` in Ubuntu, my system returns `x86_64`, and `uname -o` returns `GNU/Linux`. I would select the Linux-AMD64 binary from the https://github.com/datastax/pulsar-heartbeat/releases[releases page].
+
+Once the file is downloaded, uncompress the file with Gunzip. It will become a binary that can be executed. The filename structure is `pulsar-heartbeat-{version}-{os}-{arch}`.
```
Comment (jgillenwater) [inline modules/ROOT/pages/heartbeat-vm.adoc:11]:
```suggestion
. Use the Linux commands `uname -m` and `uname -o` to determine your process architecture.
+
For example, `uname -m` in Ubuntu might return `x86_64`, and `uname -o` returns `GNU/Linux`.

. Download the heartbeat binary `.gz` file matching your OS and process architecture from the https://github.com/datastax/pulsar-heartbeat/releases[releases page].

. Uncompress the file to be an executable binary. The filename structure is `pulsar-heartbeat-{version}-{os}-{arch}`. 
```

File: modules/ROOT/pages/heartbeat-vm.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+== Install Heartbeat Binary
+
+Download the heartbeat binary `.gz` file matching your OS and process architecture from the https://github.com/datastax/pulsar-heartbeat/releases[releases page]. Use the Linux commands `uname -m` and `uname -o` to determine your process architecture.
+
+For example, if I run the command `uname -m` in Ubuntu, my system returns `x86_64`, and `uname -o` returns `GNU/Linux`. I would select the Linux-AMD64 binary from the https://github.com/datastax/pulsar-heartbeat/releases[releases page].
+
+Once the file is downloaded, uncompress the file with Gunzip. It will become a binary that can be executed. The filename structure is `pulsar-heartbeat-{version}-{os}-{arch}`. 
+
+Here is an example:
```
Comment (jgillenwater) [inline modules/ROOT/pages/heartbeat-vm.adoc:13]:
```suggestion
Here is an example using Gunzip:
```

File: modules/ROOT/pages/heartbeat-vm.adoc [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+----
+
+== Execute Heartbeat binary
+
+The Pulsar Heartbeat configuration is defined by a `.yaml` file. A yaml template for Heartbeat is available at https://github.com/datastax/pulsar-heartbeat/blob/master/config/runtime-template.yml[].
+
+The environmental variable `PULSAR_OPS_MONITOR_CFG` tells the application where to source the file. 
+
+Run the binary file `pulsar-heartbeat-{version}-{os}-{arch}`. Here is an example of how to set up and run the application:
+[subs=attributes+]
```
Comment (jgillenwater) [inline modules/ROOT/pages/heartbeat-vm.adoc:28]:
```suggestion
The Pulsar Heartbeat configuration is defined by a `.yaml` file. A yaml template for Heartbeat is available at https://github.com/datastax/pulsar-heartbeat/blob/master/config/runtime-template.yml[]. In this file, the environmental variable `PULSAR_OPS_MONITOR_CFG` tells the application where to source the file. 

Run the binary file `pulsar-heartbeat-{version}-{os}-{arch}`.

Here is an example of how to set up and run the application:

[subs=attributes+]
```

File: modules/ROOT/pages/heartbeat-vm.adoc [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+----
+$ export PULSAR_OPS_MONITOR_CFG=/Users/username/go/src/github.com/datastax/pulsar-heartbeat/config/runtime.yml 
+
+$ ./pulsar-heartbeat-{heartbeat-version}-linux-amd64 
+2022/01/11 15:16:57  info config file /Users/username/go/src/github.com/datastax/pulsar-heartbeat/config/runtime.yml
+----
+
+== Next
+
+To install Heartbeat packaged with the Luna Streaming Helm chart, see xref:quickstart-helm-installs.adoc[Helm Chart Installation].
```
Comment (jgillenwater) [inline modules/ROOT/pages/heartbeat-vm.adoc:38]: @mendonk Is this a next step or an alternative path?

**Review by jgillenwater — APPROVED** [review]

**Pull request merged**